### ISSUE TITLE
episode_prefer_number not working as intended?

### ISSUE THREAD

**Issue by cvium** [guessit-io/guessit#532]
As per https://github.com/guessit-io/guessit/blob/master/guessit/options.py#L37-L40, it seems `episode_prefer_number` should parse `serie.213.avi` as episode 213, but what happens is this:
```
(py3) ~$ guessit -E "serie.213.avi"
For: serie.213.avi
GuessIt found: {
    "season": 213,
    "container": "avi",
    "mimetype": "video/x-msvideo",
    "type": "episode"
}
(py3) ~$ guessit --version
+-------------------------------------------------------+
+                   GuessIt 2.1.4                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

Python version: 3.6.4

It doesn't seem like your example ever worked. Here's a better example that worked _before_ 2.1.2.

2.1.1:
```
(py3) ~$ guessit --version
+-------------------------------------------------------+
+                   GuessIt 2.1.1                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
(py3) ~$ guessit "FooBar - 360 368p-Grp" -E
For: FooBar - 360 368p-Grp
GuessIt found: {
    "title": "FooBar",
    "episode": 360,
    "screen_size": "368p",
    "release_group": "Grp",
    "type": "episode"
}
```
2.1.2+:
```
(py3) ~$ guessit --version
+-------------------------------------------------------+
+                   GuessIt 2.1.2                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
(py3) ~$ guessit "FooBar - 360 368p-Grp" -E
For: FooBar - 360 368p-Grp
GuessIt found: {
    "title": "FooBar",
    "screen_size": "368p",
    "release_group": "Grp",
    "type": "movie"
}
```

**Comment by cvium** [comment]
I can see that this works properly in the `develop` branch, so I suppose this can be closed, but I'll let you (the devs) be the judge of that.

**Comment by ratoaq2** [comment]
I guess `serie.213.avi` is not a real case. I meant the name `serie`
Using `foobar.213.avi` is fine in develop branch

**Comment by cvium** [comment]
Fair enough. You should probably update the documentation to avoid the confusion.

**Comment by ratoaq2** [comment]
Fixed with #534
